test/data/example_config.yaml

```
@@ -29,6 +29,8 @@ server:
     port: 6083
     dump: true
     profiling: true
+  metrics:
+    port: 9793
```

Is there any reason to this port? If not, follow other ports.
```suggestion
    port: 6084
```

---

@WindzCUHK fixedin https://github.com/AthenZ/authorization-proxy/pull/30/commits/1a0bc8cef3624f605e59ddad85deec897964644b